templates/arm-mon-custom-alerts.json

```diff
@@ -15,16 +15,16 @@
                 "description": "Specifies whether the alert is enabled"
             }
         },
-        "resourceId": {
+        "infraResourceGroup": {
             "type": "string",
             "metadata": {
-                "description": "Resource ID of the resource emitting the metric that will be used for the comparison."
+                "description": "Infra Resource Group"
             }
         },
-        "actionGroupId": {
+        "agName": {
```

Comment:
ag can mean action group or application gateway, may be better to be more verbose

Reply:
It's in the description :)